examples/scenarioIntegrators.py

```diff
@@ -126,15 +165,17 @@ def run(show_plots, integratorCase):
         show_plots (bool): Determines if the script should display plots
         integratorCase (bool): Specify what type of integrator to use in the simulation
 
-            =======  ============================
-            String   Definition
-            =======  ============================
-            'rk4'    RK4 - default
-            'rkf45'  RKF45
-            'rkf78'  RKF78
-            'rk2'    RK2
-            'euler'  Euler or RK1
-            =======  ============================
+            =======           ============================
```

Comment:
The `=` symbols must span the width of the column text.  I sent you a correct file already on Slack.